### PR TITLE
Svg redux

### PR DESCRIPTION
**Pull request by willmcgugan**
- Change `export_svg` to write a plain SVG, without foreign object.
- Simplified SVG output, without branding.

svg_export

### CODE REVIEW

File: rich/console.py [hunk 1]
```diff
@@ -2228,274 +2223,219 @@ def export_svg(
         clear: bool = True,
         code_format: str = CONSOLE_SVG_FORMAT,
     ) -> str:
-        """Generate an SVG string from the console contents (requires record=True in Console constructor)
+        """
+        Export terminal as an SVG.
```
Comment (darrenburns) [inline rich/console.py:2227]: Think the note about `record=True` should stay, assuming it's still required.

File: tests/test_console.py [hunk 2]
```diff
@@ -493,129 +493,7 @@ def test_export_html_inline():
...
...  <g transform="translate(28, 60)">\n    <rect fill="#be3f48" x="0" y="0" width="38.2" height="27.4"/>\n    <text alignment-baseline="baseline" class="terminal-614794459-matrix" font-variant="east-asian-width-values"><tspan class="terminal-614794459-r1" x="0" y="20" textLength="37.2">foo</tspan><tspan class="terminal-614794459-r2" x="37.2" y="20" textLength="12.4">&#160;</tspan><tspan class="terminal-614794459-r2" x="49.6" y="20" textLength="62">Click</tspan><tspan class="terminal-614794459-r2" x="111.6" y="20" textLength="1128.4">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;</tspan><tspan class="terminal-614794459-r2" x="1240" y="20" textLength="12.4">\n</tspan></text>\n    </g>\n</svg>\n'
```
Comment (darrenburns) [inline tests/test_console.py:496]: Much preferred this as a multi-line string. To see what this was I had to print it and copy-paste it into an SVG file. If it's a pain I can format it if I come into this file in the future.

**Review by darrenburns — APPROVED** [review]
looks good!

File: tests/test_console.py (hunk 2 above)
Comment (willmcgugan) [reply tests/test_console.py:496]: It is a pain tbh. Tried it and I didn't produce the same string, possibly whitespace difference. Feels like something an editor could do for you.

Comment (darrenburns) [reply tests/test_console.py:496]: Cool, PyCharm handles it pretty well so I'll just do it in the future, if I remember, which I probably won't 😂

**Pull request merged**